docker/docker-compose.development.yml

```diff
@@ -50,4 +50,35 @@ services:
     environment:
       POSTGRES_USER: graph-node
       POSTGRES_PASSWORD: let-me-in
-      POSTGRES_DB: graph-node
+      POSTGRES_DB: graph-node    
+  locksmith-migration:
```

Comment:
Do we need this one too?

Reply:
the short answer is yes we need it. In production we have a two step process, migrate then start the application. We do this instead of chaining commands together to ensure that we don't end up with the application unable to server requests due to someone weird going on with a migration.

In the instance above a container will be spun up for migration and will be terminated aftewards